pkg/graph/schema.graphql

```diff
@@ -2295,7 +2287,7 @@ type Query {
   requests(after: String, first: Int!): RequestsConnection
   systemIntake(id: UUID!): SystemIntake
   systems(after: String, first: Int!): SystemConnection
-  lcidOptions: [LCIDOption!]!
+  systemIntakesWithLcids: [SystemIntake!]!
```

Comment:
Looking at the code that backs this, we could probably just call this `systemIntakes`, since `lcid` is just one of the many properties of the SystemIntake type being returned here.

(Guessing this might have been a carryover from the old `lcidOptions` name)

Reply:
Derp - I forgot that this won't return intakes without LCIDs :) Resolving!